packages/core/src/sass/app/modules/cashier.scss

```diff
@@ -43,6 +43,7 @@
             max-width: 672px;
 
             &--confirm,
+            &--no-margin,
```

Comment:
Looks like the only exception to `margin-left: 0` was the wrapper itself, now that it should be `0` too, maybe it is better to remove this styling and set `&__wrapper` to `margin: 0 auto 0 0;`?

Reply:
Hi @reza-fs it seems we need `auto` in a few situations